index.js

```diff
@@ -88,13 +88,13 @@ module.exports = function (opts) {
 
   async function parseBody(ctx) {
     if (enableJson && ((detectJSON && detectJSON(ctx)) || ctx.request.is(jsonTypes))) {
-      return await parse.json(ctx, jsonOpts);
+      return await parse.json(ctx, Object.assign({}, jsonOpts));
```

Comment:
when two requests arrive at the same time, length may be changed twice and one of the request may fail cause the wrong length value.